source/css/_objects.carousels.scss

```diff
@@ -158,3 +161,11 @@
     transform: translate(0, 0);
   }
 }
+.carousel-nav--4-3 {
```

Comment:
So this is the class that defines it as a 4:3 ratio image? Could the same thing be done to make it 16:9 images?